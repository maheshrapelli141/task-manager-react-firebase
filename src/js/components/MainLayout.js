import React from 'react';
import { HashRouter } from 'react-router-dom';

import NavBar from './NavBar';
import MainRoutes from './MainRoutes';

const MainLayout = () => (
  <HashRouter hashType="noslash">
    <div className="wrapper">
      <NavBar />
      <main>
        <div className="card">
          <div className="card-body">
            <MainRoutes />
          </div>
        </div>
      </main>
    </div>
  </HashRouter>
);

export default MainLayout;
