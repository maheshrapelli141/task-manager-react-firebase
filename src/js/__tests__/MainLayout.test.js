import React from 'react';
import { shallow } from 'enzyme';

import MainLayout from '../components/MainLayout';

const wrapper = shallow(<MainLayout />);

describe('Main layout component', () => {
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
