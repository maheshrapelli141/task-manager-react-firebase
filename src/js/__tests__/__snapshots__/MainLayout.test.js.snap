// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main layout component should render correctly 1`] = `
<HashRouter
  hashType="noslash"
>
  <div
    className="container"
  >
    <NavBar />
    <main>
      <div
        className="card"
      >
        <div
          className="card-body"
        >
          <MainRoutes />
        </div>
      </div>
    </main>
  </div>
</HashRouter>
`;
